manifests/install.pp

```diff
@@ -4,20 +4,16 @@
 class candlepin::install {
   assert_private()
 
-  $enable_pki_core = $facts['os']['release']['major'] == '8'
-
   if $candlepin::java_package {
     ensure_packages([$candlepin::java_package])
     Package[$candlepin::java_package] -> Package['candlepin']
   }
 
-  if $enable_pki_core {
-    package { 'pki-core':
-      ensure      => installed,
-      enable_only => true,
-      provider    => 'dnfmodule',
-      before      => Package['candlepin'],
-    }
+  package { 'pki-core':
```

Comment:
Thinking out loud: I wonder if this still makes sense to keep now that we have proper modularity. On the other hand: this allows the module to completely provision itself even on non-Katello environments so it's probably fine to keep.